### DISCUSSION TITLE
View not found after upgrade to version 4

### DISCUSSION THREAD

**Discussion by theDigitalViking** [laravel/nova-issues#6074] [Q&A] [answered]
- Laravel Version: 9.52.16
- Nova Version: 4.31.3
- PHP Version: 8.1.18
- Database Driver & Version: MySQL 5.7
- Operating System and Version: Mac OS Sonoma 14.0
- Browser type and version: Google Chrome 119.0.6045.159 

### Description:
I upgraded Nova from 3 to 4 using the Upgrade Guide. 
I ran into the following issue. The view [app] is not found.
As you can see on the screenshot attached.
![Bildschirmfoto 2023-11-21 um 17 47 07](https://github.com/laravel/nova-issues/assets/33465370/e69babe8-9003-42c8-bc07-9b11b21eda67)

**Comment by GautierDele** [comment]
This seems not related to Laravel Nova, you should have a look at where the named route is invoked :)

**Comment by crynobone** [comment, marked as answer]
See https://github.com/laravel/nova-issues/issues/6044#issuecomment-1812499660

**Reply by vgbrokera** [reply]
I had this same issue. I was able to fix it by adding the needed middle ware.

See the above link
